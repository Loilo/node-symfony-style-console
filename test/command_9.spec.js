

const wrapper = require('./wrapper')

test('09 Ensure that all lines are aligned to the begin of the first line in a multi-line block', wrapper(io => {
    io.block(['Custom block', 'Second custom block line'], 'CUSTOM', 'fg=white;bg=green', 'X ', true)
}))
