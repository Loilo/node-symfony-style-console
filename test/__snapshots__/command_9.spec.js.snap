// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`09 Ensure that all lines are aligned to the begin of the first line in a multi-line block 1`] = `
"X [CUSTOM] Custom block                                                         
X                                                                               
X          Second custom block line"
`;
